src/index.js

```diff
@@ -35,7 +36,7 @@ export default (
 
   onFoundSinceRevision && onFoundSinceRevision(scm.name, revision);
 
-  const rootIgnorer = createIgnorer(directory);
+  const rootIgnorer = createIgnorer(directory, ignorePath);
   const cwdIgnorer =
     currentDirectory !== directory
       ? createIgnorer(currentDirectory)
```

Comment:
May as well pass in `ignorePath` here too, in case someone is running prettier _not_ in the git root directory and has `ignorePath` set relative to a subdirectory.

```suggestion
      ? createIgnorer(currentDirectory, ignorePath)
```